roles/oVirt.external-providers/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Add external providers
-  ovirt_external_providers:
+  ovirt_external_provider:
```

Comment:
You need to replace all invocation of `ovirt_external_provider` inside the role with `ovirt_external_provider_26`, otherwise the updated module won't be used